src/prefect/tasks/aws/s3.py

```diff
@@ -46,7 +47,9 @@ def run(
             - key (str): the name of the Key within this bucket to retrieve
             - aws_credentials_secret (str, optional): the name of the Prefect Secret
                 that stores your AWS credentials; this Secret must be a JSON string
```

Comment:
Reading this closely, is it actually a "JSON string"? Or just JSON

Reply:
it really should be just JSON, but in this particular task we also convert strings to JSON automatically: https://github.com/PrefectHQ/prefect/pull/1886/files#diff-2ce447dec456df067e7517655ea99398R51-R52